docs/docsite/rst/network/user_guide/platform_index.rst

```diff
@@ -27,6 +27,7 @@ Some Ansible Network platforms support multiple connection types, privilege esca
    platform_slxos
    platform_voss
    platform_netconf_enabled
+   platform_eric_eccli
```

Comment:
Please move this new platform up in the list. The list needs to be in alphabetical order.

You also need to add your new platform to the "Settings by Platform" table in this rst file.